config/role.yaml

```diff
@@ -207,139 +208,4 @@ aggregationRule:
     # automatically.
     - matchExpressions:
         - {key: eventing.knative.dev/release, operator: Exists}
-rules: [] # Rules are automatically filled in by the controller manager.
----
-kind: ClusterRole
```

Comment:
I think you'll still need this Role (or something like it) in order for the operator to begin bootstrapping.

Did you find it was unnecessary? It would be great if we only needed the aggregated role, but I'm suspicious that we need more than that.

Reply:
The original knative-serving-operator role looked like a superset of this. I honestly haven't really checked that in depth though, I mostly eyeballed it.

Reply:
Oh, nice. that's entirely possible.

One change we had to make recently was to allow create/delete/list/watch/update for Jobs, I think you may need to readd that (as without it our upgrade tests were failing)